frontend/src/components/AccountReceiveTable.vue

```diff
@@ -53,8 +53,7 @@
         style="border-radius: 15px"
         :style="isDark ? 'color: #FFEEEE; background-color: #780A0A' : 'color: #610404; background-color: #FACDCD'"
       >
-        <i18n path="AccountReceiveTable.keys-dont-match" tag="span"
-          >{{ $t('AccountReceiveTable.keys-dont-match') }}
+        <i18n path="AccountReceiveTable.keys-dont-match" tag="span">
```

Comment:
This part looks better! Just one missing space after the period in the highlighted section

<img width="821" alt="image" src="https://user-images.githubusercontent.com/17163988/170503252-ddcc3276-a47e-4262-88c7-be04a9095ba7.png">